src/chainparams.cpp

```diff
@@ -217,10 +216,10 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = 1493596800; // May 1st 2017
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000002830dab7f76dbb7d63");
+        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000007dbe94253893cbd463");
```

Comment:
The chainwork I get at block 1354312 is `00000000000000000000000000000000000000000000006b4a51a422c1537821`. However `getblockchaininfo` reports a higher chainwork (currently at block 1356214) so I guess this is fine.